test/model/mapping/coercer_test.rb

```diff
@@ -16,22 +17,41 @@
       coercer.to_record(entity).must_equal(name: 'Tyrion Lannister')
     end
 
+    describe 'new record' do
+      # Bug:
+      # https://github.com/lotus/model/issues/155
+      it 'ignores unset values' do
+        entity = User.new(name: 'Daenerys Targaryen')
+        coercer.to_record(entity).must_equal(name: 'Daenerys Targaryen')
+      end
+
+      it 'forces nil values' do
+        entity = User.new(name: 'Daenerys Targaryen', age: nil)
+        coercer.to_record(entity).must_equal(name: 'Daenerys Targaryen')
```

Comment:
IMO it doesn't make sense send a nil value to 'database' when you are creating a entity even if you set to nil the attribute.